### PR TITLE
docs: how to make a fundamental ngx application

### PR DESCRIPTION
**Pull request by stefanoScalzo**
fixes https://github.com/SAP/fundamental-ngx/issues/76
fixes https://github.com/SAP/fundamental-ngx/issues/73

Changing the documentation due to some components being changed and new fundamental-ngx/core import

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +31,16 @@ For an existing Angular CLI application,
 
 1. **Install Fundamental-NGX.**
 
-    `ng add fundamental-ngx`
-    
-    *If you do not use the Angular CLI or if this command does not work for you, please see the [full installation guide](https://github.com/SAP/fundamental-ngx/wiki/Full-Installation-Guide).*
+    `ng add @fundamental-ngx/core`
+
+    _If you do not use the Angular CLI or if this command does not work for you, please see the [full installation guide](https://github.com/SAP/fundamental-ngx/wiki/Full-Installation-Guide)._
```
Comment (droshev) [inline README.md]: we need to update the full installation guide and explain the latest changes too

File: NEW_COMPONENT.md [hunk 2]
```diff
@@ -4,33 +4,39 @@ In this guide, we'll explore the library and documentation code base and create
...
 
 ## Create a new module in the Library
 
-First, let's generate an empty module in the library.  Each library component gets its own module, so end users can import only the modules for specific components they intend to use, if they don't want to import the entire library.
+First, let's generate an empty module in the library. Each library component gets its own module, so end users can import only the modules for specific components they intend to use, if they don't want to import the entire library.
 
-The library's source lies in the `library` directory.  From the root of the repo, cd into the library source:
+## Step1: Cd library
+
+The library's source lies in the `library` directory. From the root of the repo, cd into the library source:
```
Comment (InnaAtanasova) [inline NEW_COMPONENT.md]: we don't have `library` directory.

File: NEW_COMPONENT.md [hunk 3]
```diff
@@ -4,33 +4,39 @@ In this guide, we'll explore the library and documentation code base and create
...
 
 `cd library`
 
-We're going to be building a component called 'Poster' that displays images.  Generate a new 'poster' module in the `src/lib` directory.
+## Step2: Build module & component within libs directory
+
+We're going to be building a component called 'Poster' that displays images. Generate a new 'poster' module in the `src/lib` directory.
```
Comment (InnaAtanasova) [inline NEW_COMPONENT.md]: I don't think this is the correct path

File: NEW_COMPONENT.md (hunk 2 above)
Comment (stefanoScalzo) [reply NEW_COMPONENT.md]: libs stands for library but ill make the edit

**Comment by InnaAtanasova** [comment]
@stefanoScalzo  `Chore/how to make a fundamental ngx application` you should use the appropriate naming for the PR title.

File: README.md (hunk 1 above)
Comment (stefanoScalzo) [reply README.md]: done

File: NEW_COMPONENT.md (hunk 3 above)
Comment (stefanoScalzo) [reply NEW_COMPONENT.md]: done

File: NEW_COMPONENT.md [hunk 4]
```diff
@@ -43,20 +49,23 @@ Create an `exports` array in the poster module and add the poster component, lik
...
   selector: 'fd-poster',
 ```
 
-The documentation application is importing every component in the fundamental-ngx library module.  Open `fundamental-ngx.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
+## Step5: Add poster module to list of imports (can be in FundamentalNgxCore)
 
-We must also add `export * from './lib/poster/poster.module';` to the `fundamental-ngx/src/public_api.ts` file.  The <fd-poster> component will be an exported member of the fundamental-ngx module as well as the poster module.
+The documentation application is importing every component in the fundamental-ngx library module. Open `fundamental-ngx.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
 
+We must also add `export * from './lib/poster/poster.module';` to the `fundamental-ngx/src/public_api.ts` file. The <fd-poster> component will be an exported member of the fundamental-ngx module as well as the poster module.
```
[the remark this answers is not in the document]
Comment (stefanoScalzo) [reply NEW_COMPONENT.md]: When starting fresh sure it mentioned to not go into to many details with the api though. but isn't the export: export * from './form.module'; in the public_api.ts file

[the remark this answers is not in the document]
Comment (JKMarkowski) [reply NEW_COMPONENT.md]: Ok, actually my version is too detailed for beginning.

File: apps/docs/src/app/documentation/component-docs/poster/examples/poster-example.component.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
```
Comment (droshev) [inline apps/docs/src/app/documentation/component-docs/poster/examples/poster-example.component.ts]: The poster should not be committed.

File: libs/core/src/lib/poster/poster.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (droshev) [inline libs/core/src/lib/poster/poster.component.spec.ts]: This should be reverted

File: libs/core/src/lib/fundamental-ngx-core.module.ts [hunk 7]
```diff
@@ -1,101 +1,101 @@
...
-import {SplitButtonModule} from './split-button/split-button.module';
-import {TableModule} from './table/table.module';
-import {TabsModule} from './tabs/tabs.module';
-import {TileModule} from './tile/tile.module';
-import {TreeModule} from './tree/tree.module';
-import {TimeModule} from './time/time.module';
-import {TimePickerModule} from './time-picker/time-picker.module';
-import {ToggleModule} from './toggle/toggle.module';
-import {TokenModule} from './token/token.module';
+import { NgModule } from '@angular/core';
```
Comment (droshev) [inline libs/core/src/lib/fundamental-ngx-core.module.ts]: this file should not be touched with this PR

**Comment by droshev** [comment]
The goal fo this PR is just to update:
`NEW_COMPONENT.md`
`README.md`

File: README.md [hunk 8]
```diff
@@ -54,30 +55,42 @@ For an existing Angular CLI application,
...
 
-Run `ng test fundamental-ngx`. Append `--code-coverage` to generate code coverage documentation.
+Run `ng test @fundamental-ngx/core`. Append `--code-coverage` to generate code coverage documentation.
+
+For models prior to 0.10 use `fundamental-ngx`
 
 ## Versioning
 
-The `fundamental-ngx` library follows [Semantic Versioning](https://semver.org/). These components strictly adhere to the `[MAJOR].[MINOR].[PATCH]` numbering system (also known as `[BREAKING].[FEATURE].[FIX]`).
+The `@fundamental-ngx/core` library follows [Semantic Versioning](https://semver.org/). These components strictly adhere to the `[MAJOR].[MINOR].[PATCH]` numbering system (also known as `[BREAKING].[FEATURE].[FIX]`).
```
Comment (droshev) [inline README.md]: this needs to be reverted. We follow semantic versioning in both libraries

File: README.md [hunk 9]
```diff
@@ -54,30 +55,42 @@ For an existing Angular CLI application,
...
+
+For models prior to 0.10 use `fundamental-ngx`
```
Comment (droshev) [inline README.md]: this is not the place for that

**Pull request closed without merging**